### ISSUE TITLE
endlessly on searching for device screen

### ISSUE THREAD

**Issue by DeadNeb** [the-via/releases#85]
So the program worked the very first time I set up my keyboard but the very next day I turn it on to create some new layers and it never moves from the searching for device screen. The keyboard works fine and I can even use the keytester on VIA but it just won't allow me to configure the keyboard.

**Comment by Wilba6582** [comment]
Which keyboard/PCB?

Check if it works when not using a USB hub, i.e. directly connecting to a PC's USB port.

**Comment by DeadNeb** [comment]
yeah it's directly connected into my comps usb port but no luck...

**Comment by 1MachineElf** [comment]
I have the same problem with VIA 1.31 on Ubuntu 20.04

**Comment by dz60kid** [comment]
i have the same issue, it was working and now it isnt. it just gets stuck on searching for device.

**Comment by elliott-huge** [comment]
Has anyone managed to resolve this?

**Comment by coolkingcole** [comment]
my nk65v2 is works perfect on windows, but is not being detected on 20.04

**Comment by settsifr** [comment]
I have 3 devices plugged in (NK65, Type-9, and a CU1). They all worked fine and was able to configure them. All of sudden VIA cannot get past the "searching for devices" screen. I've tried reinstalling, restarting PC, but it is still not getting past that splash screen. Using 1.3.1

**Comment by afranke** [comment]
Same here on Fedora 34 (using the AppImage) with a Preonic.

**Comment by harryz2016** [comment]
Same here

**Comment by jimmyli56** [comment]
Same here with both an NK65 EE and a Mode65.

**Comment by usagitchi** [comment]
Same with a Portico

**Comment by jasopolis** [comment]
For some reason, on Linux it seems like the keymaps aren't automatically being loaded. Go to your vendor's website or https://github.com/the-via/keyboards/tree/master/src to download the appropriate .json, and then File > Import Keymap...
